plugins/com.google.cloud.tools.eclipse.appengine.ui/src/com/google/cloud/tools/eclipse/appengine/ui/AppEngineLibrariesSelectorGroup.java

```diff
@@ -104,87 +127,42 @@ private static String getLibraryName(Library library) {
     }
   }
 
-  private void setupDatabinding() {
-    bindingContext = new DataBindingContext(getDisplayRealm());
-    for (Button libraryButton : libraryButtons) {
-      setupDatabindingForButton(libraryButton);
-    }
+  public void dispose() {
```

Comment:
I guess I'm unclear on where this is called from then. Does it need to be public?